spec/views/events/schedule.html.erb_spec.rb

```diff
@@ -25,11 +25,19 @@
       expect(rendered).to have_selector("input[value='31.12.2017']")
     end
 
-    it 'has edit button for gameday dates' do
+    it 'has edit button for gameday dates for signed in users' do
+      @user = FactoryBot.create :user
+      @league.owner_id = @user.id
+      sign_in @user
       render
       expect(rendered).to have_selector(:link_or_button, t('events.schedule.edit_date'))
     end
 
+    it 'does not have edit button for gameday dates for signed ou users' do
+      render
+      expect(rendered).not_to have_selector(:link_or_button, t('events.schedule.edit_date'))
+    end
+
```

Comment:
Consider another test case for an user, who is signed in but has no rights to see this particular button (e.g. no event organizer rights) - this would be an ability test though. Is there already one that covers such behaviour?

Reply:
Good idea, but I don't think this must be an Ability test necessarily (the corresponeding `edit` is tested seperately). I am, however, adding a test with a signed in user with no rights to edit!